docs/src/assets/scss/docs.scss

```diff
@@ -157,3 +157,27 @@ pre[class*="language-"] {
         opacity: 1;
     }
 }
+
+.scroll-up-btn {
+    width: 50px;
+    height: 50px;
+    display: none;
+    position: fixed;
+    right: 50px;
+    bottom: 35px;
+    font-size: 1.5rem;
+    border-radius: 50%;
+    background-color: rgba(20, 20, 247, 0.5);
```

Comment:
Can we use colors to match the theme?
https://github.com/eslint/eslint/blob/75df535681d15d7d685468d637945a200301f9ee/docs/src/assets/scss/tokens/themes.scss#L116

Reply:
Yes, but instead of using the css variables for colors can i use rgba form (the way it's been already used) of theme colors so i can make them transparent a little because with opacity property the arrow icon of button is also becoming faded.